src/panel/util/EnvUtils.ts

```diff
@@ -0,0 +1,16 @@
+export const isDarkMode = (): boolean => {
+  if (process.env.BUILD_ENV === "extension") {
+    if (typeof chrome !== "undefined") {
+      return chrome.devtools.panels.themeName === "dark";
+    }
+
+    return true;
```

Comment:
```suggestion
    return chrome.devtools.panels.themeName === "dark";
```

Despite the naming, `chrome` exists within the scope of an extension/devtools for Firefox, Chrome and Edge 😅 

It's an alias for [`browser`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime#Browser_compatibility) - we should probably just move to using browser at some point (would require some testing + updating [types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/firefox-webext-browser/index.d.ts))

Reply:
Okay, that makes sense. Thanks!